core/vm/contracts.go

```diff
@@ -358,3 +374,68 @@ func (c *bn256Pairing) Run(input []byte) ([]byte, error) {
 	}
 	return false32Byte, nil
 }
+
+type blake2F struct{}
+
+func (c *blake2F) RequiredGas(input []byte) uint64 {
+	if len(input) != 213 {
```

Comment:
If input larger than `213` is used, this will throw. If that's the intended behaviour, then the EIP must also be updated to explicitly state that only exactly `213` bytes is accepted.

Reply:
Yes, my intention was to update the EIP to state it explicitly that exactly `213` bytes is accepted. I am not sure if there is any use case for passing more bytes and ignoring some of them. I personally think it's less error-prone to expect the exact length and fail when the length does not match.